ws-nextjs-app/cypress/e2e/livePage/index.cy.ts

```diff
@@ -1,28 +1,31 @@
-describe(
-  'Live Page Spec',
-  {
-    env: {
-      optionalOvveride: 'foo',
-    },
-  },
-  // @ts-expect-error Conflicts with Jest global types describe, it etc.
-  () => {
-    // eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
-    let livePageData;
-    before(() => {
-      cy.getPageData({
-        service: 'pidgin',
-        pageType: 'live',
-        id: 'c7p765ynk9qt',
-      }).then(({ body }) => {
-        livePageData = body;
-      });
+describe('Live Page Spec', () => {
+  // eslint-disable-next-line no-unused-vars, @typescript-eslint/no-unused-vars
+  let livePageData;
+  before(() => {
+    cy.getPageData({
+      service: 'pidgin',
+      pageType: 'live',
+      id: 'c7p765ynk9qt',
+    }).then(({ body }) => {
+      livePageData = body;
+    });
+  });
+
+  if (Cypress.env('APP_ENV') === 'test') {
+    it('visits page and passes', () => {
+      cy.visit('/pidgin/live/c7p765ynk9qt');
     });
 
-    if (Cypress.env('APP_ENV') === 'test') {
-      it('passes', () => {
+    describe(`SEO Metadata`, () => {
+      it('should include mainEntityOfPage in the LinkedData', () => {
         cy.visit('/pidgin/live/c7p765ynk9qt');
+
+        cy.get('script[type="application/ld+json"]')
+          .should('contain', 'mainEntityOfPage')
+          .and('contain', 'author')
+          .and('contain', 'headline')
+          .and('contain', 'liveBlogPosting');
       });
-    }
-  },
-);
+    });
+  }
+});
```

Comment:
Yea its a good question. Tbh, I just added this in here particularly because we did ages ago for Article Pages but also just to make a start into adding more e2es for the Next app.

Can be removed though as you're right, they aren't really 'e2e' in nature.

Reply:
Yeah makes sense, I guess it just puts a question when we come back to the e2e test file and see this test it might cause a bit of confusion as what we should have as our e2e tests. I still don't know fully how we've defined our e2e test strategy, but that opens up another conversation to be had

Reply:
Its a fair point, I'll remove it for now sure.